### ISSUE TITLE
Backup - save file without password

### ISSUE THREAD

**Issue by borodadada** [Authenticator-Extension/authenticator.cc#44]
backup save file without password not working, last version google chrome + this auth app for chrome
I added new site in authenticator and push save backup, after i open editor, backup file no have new url, old version always work, fix please!

**Comment by tongduanmingo** [comment]
same problem is with me too any solution ??

**Comment by borodadada** [comment]
no, the developer answered in another thread, the result is 0, as if this is some kind of unofficial version, and there is no official version of the plugin
